CHANGELOG.md

```diff
@@ -15,6 +15,8 @@
 - Fix: [#2725] Currency preference selection shows invalid data.
 - Fix: [#2727] Bankruptcy warnings do not appear.
 - Fix: [#2735] Map generator does not set the season on trees.
+- Fix: [#2742] Sound effects playing when title screen is paused.
+- Fix: [#2743] Title music not playing while scenario is loading.
```

Comment:
Generally I'd prefer if you did one fix/change per PR. It keeps things separated and makes it easy if we ever have to `git revert` something. For now though, I'm happy to leave this PR as is